source/administration/mmctl-cli-tool.rst

```diff
@@ -0,0 +1,2320 @@
+mmctl Command Line Tool (Beta)
+==============================
+
+The mmctl tool is a remote CLI tool for Mattermost which is installed locally and uses the Mattermost API. Authentication
+is done with either login credentials or an authentication token. <elaborate more on this, provide links to other docs>.
+
+Being installed locally allows a System Admin to run CLI commands even in instances where there is no access to the
+server (e.g., via SSH). This tool is currently in beta and can be used alongside the Mattermost CLI tool.
+In the future, the Mattermost CLI tool will be deprecated.
+
+**Notes**
+
+ -  Parameters in CLI commands are order-specific.
+ -  If special characters (``!``, ``|``, ``(``, ``)``, ``\``, ``'``, and ``"``) are used, the entire argument needs to be surrounded
```

Comment:
Some of these bullet points need formatting tweaks.

Reply:
Thanks! Fixed.